### ISSUE TITLE
【Train issue with caffe2 detectron】Aborted at 1523501813 (unix time) try "date -d @1523501813" if you are using GNU date *** PC: @     0x7fa35103733a (unknown)

### ISSUE THREAD

**Issue by Shentongmo** [pytorch/pytorch#6542]
I try to run resnet50_trainer.py to build my model by using 100 images, but I get the following error messages. My train method is drawn on the following web, https://github.com/royhuang9/Detectron. How to fix the error and run correctly using Caffe2? I am not sure if this is a Caffe2 issue. 

Train command:
python2 tools/train_net.py --cfg experiments/e2e_faster_rcnn_resnet-50-FPN_pascal2007.yaml  OUTPUT_DIR experiments/output  

Error meassage:
json_stats: {"accuracy_cls": 1.000000, "eta": "1:59:55", "iter": 26200, "loss": 0.000865, "loss_bbox": 0.000183, "loss_cls": 0.000680, "loss_rpn_bbox_fpn2": 0.000000, "loss_rpn_bbox_fpn3": 0.000000, "loss_rpn_bbox_fpn4": 0.000003, "loss_rpn_bbox_fpn5": 0.000003, "loss_rpn_bbox_fpn6": 0.000000, "loss_rpn_cls_fpn2": 0.000000, "loss_rpn_cls_fpn3": 0.000005, "loss_rpn_cls_fpn4": 0.000004, "loss_rpn_cls_fpn5": 0.000002, "loss_rpn_cls_fpn6": 0.000000, "lr": 0.002500, "mb_qsize": 64, "mem": 2451, "time": 0.212877}
*** Aborted at 1523501813 (unix time) try "date -d @1523501813" if you are using GNU date ***
PC: @     0x7fa35103733a (unknown)
*** SIGSEGV (@0x0) received by PID 6778 (TID 0x7fa2edffb700) from PID 0; stack trace: ***
    @     0x7fa351390390 (unknown)
    @     0x7fa35103733a (unknown)
    @     0x7fa351039184 __libc_malloc
    @     0x7fa348101c51 (unknown)
    @     0x7fa348164232 (unknown)
    @     0x7fa3481642ea (unknown)
    @     0x7fa3481f29d4 (unknown)
    @     0x7fa3481f2da0 (unknown)
    @           0x4c195b PyEval_EvalFrameEx
    @           0x4c136f PyEval_EvalFrameEx
    @           0x4b9ab6 PyEval_EvalCodeEx
    @           0x4d54b9 (unknown)
    @           0x4eebee (unknown)
    @           0x4a577e PyObject_Call
    @           0x4c5e10 PyEval_CallObjectWithKeywords
    @     0x7fa33adca3d0 pybind11::detail::object_api<>::operator()<>()
    @     0x7fa33adcbb51 caffe2::python::PythonOpBase<>::RunOnDevice()
    @     0x7fa33ad932cb caffe2::Operator<>::Run()
    @     0x7fa33addce3a _ZN6caffe213GPUFallbackOpINS_6python8PythonOpINS_10CPUContextELb0EEENS_11SkipIndicesIJEEEE11RunOnDeviceEv
    @     0x7fa33add95d5 caffe2::Operator<>::Run()
    @     0x7fa336bcef0a caffe2::DAGNet::RunAt()
    @     0x7fa336bcdbbe caffe2::DAGNetBase::WorkerFunction()
    @     0x7fa34a5cbc80 (unknown)
    @     0x7fa3513866ba start_thread
    @     0x7fa3510bc41d clone
    @                0x0 (unknown)

**Comment by sefira** [comment]
I meet the same error...

**Comment by Shentongmo** [comment]
I have fixed the problem. I think you should change the suitable value of batch_size or max_iter. Have a try.

**Comment by sefira** [comment]
@Shentongmo Thanks for your reply!

What do you mean by saying “the suitable value of batch_size or max_iter”?
I run the defualt config tutorial_2gpu_e2e_faster_rcnn_R-50-FPN.yaml in COCO17 dataset, where the MAX_ITER: 30000 BATCH_SIZE_PER_IM: 256 IMS_PER_BATCH: 2

Do you have a heuristic method to choose “the suitable value of batch_size or max_iter”? What is the exactly problem bought by batch_size and max_iter in your try?

Thanks again~
